Typescript client examples have a version issue

i installed the packages:
```
$ npm install @foxglove/ws-protocol
$ npm install ws
```
and added some code:
```
  import { FoxgloveClient } from "@foxglove/ws-protocol";
  import WebSocket from "ws";

  const address = 'ws://localhost:8765'

  const client = new FoxgloveClient({
    ws: new WebSocket(address, [FoxgloveClient.SUPPORTED_SUBPROTOCOL]),
  });
  ```
but im getting a compiler error on IWebSocket:
```
$ ./node_modules/.bin/tsc
src/main.ts:21:5 - error TS2322: Type 'WebSocket' is not assignable to type 'IWebSocket'.
  Types of property 'send' are incompatible.
    Type '{ (data: BufferLike, cb?: ((err?: Error | undefined) => void) | undefined): void; (data: BufferLike, options: { mask?: boolean | undefined; binary?: boolean | undefined; compress?: boolean | undefined; fin?: boolean | undefined; }, cb?: ((err?: Error | undefined) => void) | undefined): void; }' is not assignable to type '(data: string | ArrayBuffer | ArrayBufferView | Blob, options?: { fin?: boolean | undefined; } | undefined) => void'.
      Types of parameters 'data' and 'data' are incompatible.
        Type 'string | ArrayBuffer | ArrayBufferView | Blob' is not assignable to type 'BufferLike'.
          Type 'Blob' is not assignable to type 'BufferLike'.
            Type 'Blob' is missing the following properties from type 'Buffer': write, toJSON, equals, compare, and 95 more.

21     ws: new WebSocket(address, [FoxgloveClient.SUPPORTED_SUBPROTOCOL]),
       ~~

  node_modules/@foxglove/ws-protocol/dist/esm/src/FoxgloveClient.d.ts:23:9
    23         ws: IWebSocket;
               ~~
    The expected type comes from property 'ws' which is declared here on type '{ ws: IWebSocket; }'


Found 1 error in src/main.ts:21
```

I've tried using the same ws version as the example, but the example protocol library is very out of date.
An up to date working example would be greatly appreciated!

--- a/typescript/ws-protocol-examples/package.json
+++ b/typescript/ws-protocol-examples/package.json
@@ -46,7 +46,7 @@
     "@types/debug": "^4.1.7",
     "@types/lodash": "^4.14.191",
     "@types/node": "^18.11.18",
-    "@types/ws": "^8.5.3",
+    "@types/ws": "^8.5.4",
     "@typescript-eslint/eslint-plugin": "5.48.1",
     "@typescript-eslint/parser": "5.48.1",
     "eslint": "8.31.0",

--- a/typescript/ws-protocol-examples/package.json
+++ b/typescript/ws-protocol-examples/package.json
@@ -64,7 +64,7 @@
     "@foxglove/rosmsg-serialization": "^1.5.3",
     "@foxglove/rosmsg2-serialization": "^1.1.1",
     "@foxglove/rostime": "^1.1.2",
-    "@foxglove/ws-protocol": "^0.3.1",
+    "@foxglove/ws-protocol": "^0.3.2",
     "@mcap/core": "^0.3.0",
     "boxen": "^7.0.1",
     "commander": "^9.5.0",

--- a/typescript/ws-protocol/package.json
+++ b/typescript/ws-protocol/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@foxglove/ws-protocol",
-  "version": "0.3.1",
+  "version": "0.3.2",
   "description": "Foxglove Studio WebSocket protocol",
   "keywords": [
     "foxglove",

--- a/typescript/ws-protocol/src/FoxgloveServer.ts
+++ b/typescript/ws-protocol/src/FoxgloveServer.ts
@@ -526,8 +526,6 @@ export default class FoxgloveServer {
     if (connection.send.length > 1) {
       connection.send(header.buffer, { fin: false });
       connection.send(payload, { fin: true });
-    } else if (typeof Blob === "function") {
-      connection.send(new Blob([header.buffer, payload]));
     } else {
       const buffer = new Uint8Array(header.buffer.byteLength + payload.byteLength);
       buffer.set(new Uint8Array(header.buffer), 0);

--- a/typescript/ws-protocol/src/types.ts
+++ b/typescript/ws-protocol/src/types.ts
@@ -148,7 +148,7 @@ export interface IWebSocket {
   onmessage: ((event: any) => void) | null | undefined; // eslint-disable-line @typescript-eslint/no-explicit-any
   close(): void;
   send(
-    data: string | ArrayBuffer | Blob | ArrayBufferView,
+    data: string | ArrayBuffer | ArrayBufferView,
     /** Options available in Node "ws" library */
     options?: { fin?: boolean },
   ): void;

